### PR TITLE
[FEAT/PAYMENT + GUIDE] Create Payment & Policy Page

### PR DESCRIPTION
**Pull request by NguyenTriBaoThang**
## Description

- Create Payment, Guide

## Project

- [x] Website
- [ ] API
- [ ] Documentation
- [ ] Other... (please describe)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other... (please describe)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B
- [x] No tests needed

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Other... (please describe)

### CODE REVIEW

File: apps/website/src/features/Payment/index.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
+import Container from '@mui/material/Container';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import Typography from '@mui/material/Typography';
+import NavigateNextIcon from '@mui/icons-material/NavigateNext';
+import useMetadata from '../../hooks/useMetadata';
+import bg from '../../assets/images/banners/bg.svg';
```
Comment (qoucname2202) [inline apps/website/src/features/Payment/index.tsx:9]: @NguyenTriBaoThang Format link image

File: apps/website/src/features/Payment/index.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  imgSrc: string;
+}
+
+const CustomLink: React.FC<CustomLinkProps> = ({ to, text, imgSrc }) => {
+  return (
+    <Link href={to} sx={{ display: 'table', width: '100%', marginBottom: '20px', textDecoration: 'none' }}>
+      <Box sx={{ display: 'table-cell', width: '36px', verticalAlign: 'middle' }}>
+        <img src={imgSrc} alt="icon" style={{ color: '#67A99F' }} />
+      </Box>
+      <Typography variant="body1" sx={{
```
Comment (qoucname2202) [inline apps/website/src/features/Payment/index.tsx:40]: @NguyenTriBaoThang Using className properties tailwindcss

**Review by qoucname2202 — CHANGES_REQUESTED** [review]
@NguyenTriBaoThang Fix code style for each file has comment

File: apps/website/src/features/Payment/index.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  to: string;
+  text: string;
+  imgSrc: string;
+}
+
+const CustomLink: React.FC<CustomLinkProps> = ({ to, text, imgSrc }) => {
+  return (
+    <Link href={to} sx={{ display: 'table', width: '100%', marginBottom: '20px', textDecoration: 'none' }}>
+      <Box sx={{ display: 'table-cell', width: '36px', verticalAlign: 'middle' }}>
+        <img src={imgSrc} alt="icon" style={{ color: '#67A99F' }} />
```
Comment (foxminchan) [inline apps/website/src/features/Payment/index.tsx:38]: @NguyenTriBaoThang Image must be have loading lazy

**Docs:** [here](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading)

**Review by foxminchan — CHANGES_REQUESTED** [review]

File: apps/website/src/features/Payment/index.tsx [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  title: string;
+};
+
+interface CustomLinkProps {
+  to: string;
+  text: string;
+  imgSrc: string;
+}
+
+const CustomLink: React.FC<CustomLinkProps> = ({ to, text, imgSrc }) => {
```
Comment (foxminchan) [inline apps/website/src/features/Payment/index.tsx:34]: @NguyenTriBaoThang use tsrfc

File: apps/website/src/features/Policy/index.tsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+interface CustomListItemProps {
+  text: string; 
+}
+
+const GrayDivider = () => {
+  return <Divider sx={{ backgroundColor: '#ccc', height: '1px', margin: '10px 0px' }} />;
+}; 
+
+const CustomListItem: React.FC<CustomListItemProps> = ({ text }) => {
```
Comment (foxminchan) [inline apps/website/src/features/Policy/index.tsx:24]: @NguyenTriBaoThang use tsrfc

**Review by foxminchan — CHANGES_REQUESTED** [review]

**Review by qoucname2202 — APPROVED** [review]
@foxminchan Approve merger

File: apps/website/src/features/Policy/policy.data.ts [hunk 6]
```diff
@@ -0,0 +1,78 @@
+export const Section_1 = [
```
Comment (foxminchan) [inline apps/website/src/features/Policy/policy.data.ts:1]: @NguyenTriBaoThang Meaningless variable names

File: apps/website/src/features/Policy/policy.data.ts [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    {
+        name: 'Mọi trường hợp sử dụng lại các thông tin, nội dung trên Cổng Dịch vụ công Quốc gia để đăng tải trên các phương tiện truyền thông phải thực hiện theo quy định của pháp luật.',
+    },
+    
+    {
+        name: 'Đồng ý chia sẻ thông tin đăng ký trên Cổng Dịch vụ công Quốc gia theo quy định của Cổng Dịch vụ công Quốc gia.',
+    },
+];
+
+export const Section_2 = [
```
Comment (foxminchan) [inline apps/website/src/features/Policy/policy.data.ts:23]: @NguyenTriBaoThang Meaningless variable names

File: apps/website/src/features/Policy/policy.data.ts [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    {
+        name: 'Gian lận, chiếm đoạt hoặc sử dụng trái phép chữ ký điện tử của người khác; sửa chữa, thay đổi hoặc công khai hóa thông tin riêng hợp pháp của cơ quan, tổ chức, cá nhân trên Cổng mà không được phép của chủ sở hữu thông tin đó.',
+    },
+
+    {
+        name: 'Các hành vi khác theo quy định của pháp luật.',
+    },
+];
+
+export const Section_4 = [
```
Comment (foxminchan) [inline apps/website/src/features/Policy/policy.data.ts:57]: @NguyenTriBaoThang Meaningless variable names

File: apps/website/src/features/Policy/index.tsx
Comment (foxminchan) [inline apps/website/src/features/Policy/index.tsx:1]: @NguyenTriBaoThang make sure you have install prettier for format code

**Tools:** [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)

**Review by foxminchan — CHANGES_REQUESTED** [review]
@NguyenTriBaoThang resolve coflict, fix code smell in sonarcloud and file has comment

**Review by foxminchan — APPROVED** [review]

**Pull request merged**